utility/utils.py

```diff
@@ -702,9 +705,10 @@ def create_html_file(test_result) -> str:
     except KeyError as kerr:
         log.error(f"Key not found : {kerr}")
         exit(1)
-
-    log_link = "http://magna002.ceph.redhat.com/cephci-jenkins/{run}/".format(
-        run=run_name
+    log_link = (
```

Comment:
I believe we can attempt to do string concatenation.

Maybe also add a comment on why we are looking for '/ceph/cephci-jenins' in the `run_dir` variable.